CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### Added
 
 - Added support for the Steam authentication flow.
+- `CopySchema` now scans for nested package dependencies.
```

Comment:
This is probably in `Fixed` - as this is really a bug (imo)

Reply:
Also this message isn't very useful to a user:

```
Fixed a bug where Code Generation would not capture schema in a nested package dependency
```

or something similar, (and better pls)